src/Components/Footers/Local.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Northeastern\Blade\Components\Footers;
+
+use Illuminate\Support\Facades\View;
+use Illuminate\View\Component;
+
+class Local extends Component
+{
+    public $links;
+
+    public $logoHref;
+
+    public $facebookUrl;
+
+    public $twitterUrl;
+
+    public $youtubeUrl;
+
+    public $linkedinUrl;
+
+    public $instagramUrl;
+
+    public $snapchatUrl;
+
+    public function __construct(
+        $links = [],
+        $logoHref = '#',
+        $facebookUrl = null,
+        $twitterUrl = null,
+        $youtubeUrl = null,
+        $linkedinUrl = null,
+        $instagramUrl = null,
+        $snapchatUrl = null
+    ) {
+        $this->links = $links;
+
+        $this->logoHref = $logoHref;
+
+        $this->facebookUrl = $facebookUrl;
+        $this->twitterUrl = $twitterUrl;
+        $this->youtubeUrl = $youtubeUrl;
+        $this->linkedinUrl = $linkedinUrl;
+        $this->instagramUrl = $instagramUrl;
+        $this->snapchatUrl = $snapchatUrl;
```

Comment:
Do you think these should all be `${something}Url` or `${something}Href` for consistency? The `$logoHref` sticks out to me here. Maybe `$logoHref` should just be `$logoUrl`?

Reply:
Yes. We can do `$logoUrl` for consistency. Sounds good.